docs/src/app/experiments/rtl.tsx

```diff
@@ -0,0 +1,202 @@
+                            onClick={createHandleMenuClick('Style/Heading/Level 1')}
+                          >
+                            Level 1
+                          </Menu.Item>
+                          <Menu.Item
+                            className={c.item}
+                            onClick={createHandleMenuClick('Style/Heading/Level 2')}
+                          >
+                            Level 2
+                          </Menu.Item>
+                          <Menu.Item
+                            className={c.item}
+                            onClick={createHandleMenuClick('Style/Heading/Level 3')}
+                          >
+                            Level 3
+                          </Menu.Item>
+                        </Menu.Popup>
+                      </Menu.Positioner>
+                    </Menu.Root>
+                    <Menu.Item
+                      className={c.item}
+                      onClick={createHandleMenuClick('Style/Paragraph')}
+                    >
+                      Paragraph
+                    </Menu.Item>
+                    <Menu.Root>
+                      <Menu.SubmenuTrigger className={c.submenutrigger}>
+                        List
+                      </Menu.SubmenuTrigger>
+                      <Menu.Positioner
+                        align="start"
+                        side="inline-end"
+                        className={c.positioner}
+                        dir={dir}
+                      >
+                        <Menu.Popup className={c.popup}>
+                          <Menu.Item
+                            className={c.item}
+                            onClick={createHandleMenuClick('Style/List/Ordered')}
+                          >
+                            Ordered
+                          </Menu.Item>
+                          <Menu.Item
+                            className={c.item}
+                            onClick={createHandleMenuClick('Style/List/Unordered')}
+                          >
+                            Unordered
+                          </Menu.Item>
+                        </Menu.Popup>
+                      </Menu.Positioner>
+                    </Menu.Root>
+                  </Menu.Popup>
+                </Menu.Positioner>
+              </Menu.Root>
+
+              <Menu.Item
+                className={c.item}
+                onClick={createHandleMenuClick('Clear formatting')}
+              >
+                Clear formatting
+              </Menu.Item>
+            </Menu.Popup>
+          </Menu.Positioner>
+        </Menu.Root>
+
+        <PreviewCard.Root>
+          <PreviewCard.Trigger href="#" className={c.trigger}>
+            PreviewCard.Trigger
+          </PreviewCard.Trigger>
+          <PreviewCard.Positioner
+            sideOffset={8}
+            side="inline-end"
+            align="center"
+            className={c.positioner}
+            dir={dir}
```

Comment:
The only quirk is if you don't put `dir` on `html`, you need to put it on all the positioners manually because of portaling